spring-cloud-generator/src/main/java/com/google/cloud/generator/spring/composer/SpringAutoConfigClassComposer.java

```diff
@@ -387,7 +385,8 @@ private static MethodDefinition createTransportChannelProviderBeanMethod(
             .setReturnType(STATIC_TYPES.get("TransportChannelProvider"))
             .build();
 
-    if (hasRestOption) {
+    // For GRPC+REST libraries, choose provider according to configuration property
+    if (ComposerUtils.shouldSupportRestOptionWithGrpcDefault(transport, service)) {
       //      if (this.clientProperties.isUseRest()) {
```

Comment:
Can this commented out code be removed?
(Also other places in this file.)

Reply:
I would add additional comments instead of the code.

Reply:
I would prefer to remove them or add more comment to indicate this commented out code is a sample of generated code, otherwise developers that are not familiar with the codebase can easily get confused. Another thing you can try is only adding the sample of generated code on top of the method, so it's less likely to get confused.

Reply:
Makes sense, I see the source for confusion now. Cleaned up the comments in https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1849/commits/2628a462e8f5a00761231e2d3552be661f356ee3.